File: app/src/main/java/com/cartenz/cartenztaxcore/api/ApiInterface.java
package com.cartenz.cartenztaxcore.api;


import com.cartenz.cartenztaxcore.api.dao.LoginDao;

import okhttp3.RequestBody;
import retrofit2.http.Body;
import retrofit2.http.POST;
import rx.Observable;

public interface ApiInterface {

    //post
    @POST("login")
    Observable<LoginDao> login(
            @Body RequestBody body
    );


}

File: kotlinapp/src/main/java/com/cartenz/kotlinapp/api/ApiInterface.kt
package com.cartenz.kotlinapp.api

import com.cartenz.kotlinapp.api.dao.LoginDao
import com.cartenz.kotlinapp.api.dao.SimpleStringDao
import okhttp3.RequestBody
import retrofit2.http.Body
import retrofit2.http.POST
import rx.Observable

interface ApiInterface {

    //post
    @POST("login")
    fun login(
            @Body body: RequestBody
    ): Observable<LoginDao>

    @POST("forgot")
    fun forgotpass(
            @Body body: RequestBody
    ): Observable<SimpleStringDao>

}
